### PR TITLE
[zh-cn]: create the translation of the MathML element `<mpadded>`

### PR DESCRIPTION
**Pull request by tianyeeT**
### Related issues and pull requests
#20746

### CODE REVIEW

**Review by yin1999 — CHANGES_REQUESTED** [review]
和 #20847 类似，这些问题请先自己处理一下。还有其他的几个 PR 也是，请先修正（我先标记为“草稿”了，修好后再转“可供审阅”）。

File: files/zh-cn/web/mathml/element/mpadded/index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: <mpadded>
+slug: Web/MathML/Element/mpadded
+l10n:
+  sourceCommit: 4d5e2c11f4b8cc32e54d2527d9576ed26ced9458
+---
+
+{{MathMLRef}}
+
+**`<mpadded>`** 这个 [MathML](/zh-CN/docs/Web/MathML) 元素用于添加额外的填充，并设置封闭内容的位置和大小的一般调整。
```
Comment (yin1999) [inline files/zh-cn/web/mathml/element/mpadded/index.md:10]: padding 与 [CSS padding 属性](https://developer.mozilla.org/zh-CN/docs/Web/CSS/padding)对应，翻译为“内边距”吧（然后把“这个”去掉）

```suggestion
**`<mpadded>`** [MathML](/zh-CN/docs/Web/MathML) 元素用于添加额外的内边距，并设置封闭内容的位置和大小的一般调整。
```

File: files/zh-cn/web/mathml/element/mpadded/index.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+### 旧版语法
+
+对于 `depth`、`height`、`lspace`、`voffset` 和 `width` 属性，一些浏览器可能接受更复杂的语法：
+
+1. 可选的 `+` 或 `-` 符号作为前缀，指定对应维度的增量或减量（如果缺失，则对应维度直接设置为指定值）。
+2. 后跟一个 [`<unsigned-number>`](/zh-CN/docs/Web/MathML/Values#mathml_特定类型)（下面称为 α）。
+3. 可选地跟随一个值（如果缺失，则指定值被解释为“α 的百分比的 100 倍”）。
+   - 一个[unit](/zh-CN/docs/Web/MathML/Values#单位)。指定值的解释方式与旧版 MathML 长度相同。
+   - 一个[namedspace constant](/zh-CN/docs/Web/MathML/Values#常数)。指定值被解释为 α 乘以常量。
```
Comment (yin1999) [inline files/zh-cn/web/mathml/element/mpadded/index.md:35]:
```suggestion
   - 一个[单位](/zh-CN/docs/Web/MathML/Values#单位)。指定值的解释方式与旧版 MathML 长度相同。
   - 一个[命名空间常量](/zh-CN/docs/Web/MathML/Values#常数)。指定值被解释为 α 乘以常量。
```

File: files/zh-cn/web/mathml/element/mpadded/index.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+           lspace="300px" voffset="-2em"
+           style="background: lightblue">
+    <mi>x</mi>
+    <mo>+</mo>
+    <mi>y</mi>
+  </mpadded>
+</math>
+```
+
+{{ EmbedLiveSample('示例', 700, 200) }}
```
Comment (yin1999) [inline files/zh-cn/web/mathml/element/mpadded/index.md:55]: 有没有发现两个示例重叠在一起了：

image

这就是我们为什么要强调[“注意一并修改运行实例宏的第一个参数为相对应的中文标题”](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E7%BF%BB%E8%AF%91%E7%A4%BA%E4%BE%8B%E4%BB%A3%E7%A0%81%E5%9D%97)的原因。

```suggestion
{{ EmbedLiveSample('尺寸和偏移', 700, 200) }}
```

File: files/zh-cn/web/mathml/element/mpadded/index.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  </mpadded>
+
+  <!-- 递减至命名空间的倍数 -->
+  <mpadded width="-1thickmathspace" style="background: pink">
+    <mtext>-.5thickmathspace</mtext>
+  </mpadded>
+</math>
+```
+
+{{ EmbedLiveSample('示例', 700, 200) }}
```
Comment (yin1999) [inline files/zh-cn/web/mathml/element/mpadded/index.md:83]:
```suggestion
{{ EmbedLiveSample('旧版语法', 700, 200) }}
```

File: files/zh-cn/web/mathml/element/mpadded/index.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+### 旧版语法
+
+```html
+<math display="block">
+  <!-- 以一个长度增加 -->
```
Comment (yin1999) [inline files/zh-cn/web/mathml/element/mpadded/index.md:61]:
```suggestion
  <!-- 增加一定的长度 -->
```

File: files/zh-cn/web/mathml/element/mpadded/index.md (hunk 3 above)
Comment (tianyeeT) [reply files/zh-cn/web/mathml/element/mpadded/index.md:55]: 谢谢解答。明白了，以后在翻译的时候会注意。重叠的原因是因为我把两个名字都叫做”示例“导致的嘛。

Comment (tianyeeT) [reply files/zh-cn/web/mathml/element/mpadded/index.md]: 我懂了，它渲染拿代码是根据那一章的标题拿的。

**Review by yin1999 — APPROVED** [review]

File: files/zh-cn/web/mathml/element/mpadded/index.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+  </mpadded>
+
+  <!-- 递减至命名空间的倍数 -->
+  <mpadded width="-1thickmathspace" style="background: pink">
+    <mtext>-.5thickmathspace</mtext>
+  </mpadded>
+</math>
+```
+
+{{ EmbedLiveSample('旧版语法', 700, 200) }}
```
Comment (yin1999) [inline files/zh-cn/web/mathml/element/mpadded/index.md:83]:
```suggestion
{{ EmbedLiveSample('旧版语法_2', 700, 200) }}
```

**Review by mo7mda7 — COMMENTED** [review]
<span style="caret-color: rgb(214, 114, 32); color: rgb(214, 114, 32); font-family: arial, helvetica, clean, sans-serif; font-size: 14px; font-style: normal; font-variant-caps: normal; font-weight: 700; letter-spacing: normal; orphans: auto; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-tap-highlight-color: rgba(26, 26, 26, 0.3); -webkit-text-size-adjust: 100%; -webkit-text-stroke-width: 0px; background-color: rgb(255, 242, 231); text-decoration: none; display: inline !important; float: none;">Invalid Login Credentials</span>

The CSS code you provided seems to be a styling for displaying the text "Invalid Login Credentials" in a specific style. However, it is important to note that this code snippet alone does not handle the functionality of validating login credentials. 

To handle the validation of login credentials, you would typically use JavaScript along with CSS. JavaScript can be used to check the entered credentials against a database or perform other validation logic. CSS can then be used to style the form elements based on the validation results.

If you are looking for CSS pseudo-classes related to form validation, there are a few options available. One such pseudo-class is `:invalid`, which represents form elements or input fields that have failed validation [[1]](https://developer.mozilla.org/ja/docs/Web/CSS/:invalid). You can use this pseudo-class to apply specific styles to invalid input fields, such as changing the background color to pink.

Here's an example of how you can use the `:invalid` pseudo-class to style invalid input fields:

```css
input:invalid {
  background-color: pink;
}
```

This CSS rule will apply the specified background color to any invalid input field. You can customize the styles according to your needs.

It's worth mentioning that the `:invalid` pseudo-class is just one of the pseudo-classes related to form validation. There are other pseudo-classes like `:required`, `:optional`, and `:valid` that you can use to style form elements based on their validation status [[1]](https://developer.mozilla.org/ja/docs/Web/CSS/:invalid).

---
Learn more:
1. [:invalid - CSS: カスケーディングスタイルシート | MDN](https://developer.mozilla.org/ja/docs/Web/CSS/:invalid)
2. [:user-invalid (:-moz-ui-invalid) - CSS: カスケーディングスタイルシート | MDN](https://developer.mozilla.org/ja/docs/Web/CSS/:user-invalid)
3. [:user-invalidの魅力を伝えたい](https://zenn.dev/kyoheyee/articles/5867b2389ac635)

**Pull request merged**